cmd/install/addon/cmd.go

```diff
@@ -242,8 +268,51 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	billingModel := args.billingModel
+	billingModelAccountID := args.billingModelAccountID
+
+	if !cmd.Flags().Changed(billingModelFlag) && !interactive.Enabled() {
+		interactive.Enabled()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && !cmd.Flags().Changed("billing-model") {
```

Comment:
Did you mean to leave "billing-model" hardcoded here instead of using billingModelFlag?